neo/Cryptography/MerkleTree.cs

```diff
@@ -14,7 +14,7 @@ public class MerkleTree
 
         internal MerkleTree(UInt256[] hashes)
         {
-            if (hashes.Length == 0) throw new ArgumentException();
+            if (hashes.Length == 0 || hashes == null) throw new ArgumentException();
```

Comment:
Oh, and by the way, this check should probably be done the other way around: `hashes == null || hashes.Length == 0`.